src/plugins/kibana_usage_collection/server/collectors/application_usage/rollups.ts

```diff
@@ -125,7 +123,11 @@ async function getDailyDoc(
   dayId: string
 ): Promise<ApplicationUsageDailyWithVersion> {
   try {
-    return await savedObjectsClient.get<ApplicationUsageDaily>(SAVED_OBJECTS_DAILY_TYPE, id);
+    const { attributes, version } = await savedObjectsClient.get<ApplicationUsageDaily>(
+      SAVED_OBJECTS_DAILY_TYPE,
+      id
+    );
+    return { attributes, version };
```

Comment:
`ApplicationUsageDailyWithVersion` only defines `attributes` and `version`, however we were returning the whole saved object. it has effectively no impact as we were overriding the other properties in the calling code, however this could be dangerous in case of refactoring, so I filtered the returning properties.